ci/jenkins/test.sh

```diff
@@ -380,24 +380,18 @@ function deliver_antrea_linux {
 
     echo "==== Start building and delivering Linux Docker images ===="
     DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --pull
-    if [[ "$TESTCASE" == "windows-networkpolicy-process" ]]; then
-        make windows-bin
-    fi
 
     echo "====== Delivering Antrea to all Nodes ======"
     export_govc_env_var
 
-    # Enable verbose log for troubleshooting.
-    sed -i "s/--v=0/--v=4/g" build/yamls/antrea-windows.yml
-
-    if [[ "${PROXY_ALL}" == false && ${TESTCASE} =~ "windows-e2e" ]]; then
-        sed -i "s|.*proxyAll: true|      proxyAll: false|g" build/yamls/antrea.yml build/yamls/antrea-windows.yml
+    if [[ "${PROXY_ALL}" == false ]]; then
+        sed -i "s|.*proxyAll: true|      proxyAll: false|g" build/yamls/antrea.yml
     else
         echo "====== Updating yaml files to enable proxyAll ======"
         KUBERNETES_SVC_EP_IP=$(kubectl get endpoints kubernetes -o jsonpath='{.subsets[0].addresses[0].ip}')
         KUBERNETES_SVC_EP_PORT=$(kubectl get endpoints kubernetes -o jsonpath='{.subsets[0].ports[0].port}')
         KUBERNETES_SVC_EP_ADDR="${KUBERNETES_SVC_EP_IP}:${KUBERNETES_SVC_EP_PORT}"
-        sed -i "s|.*kubeAPIServerOverride: \"\"|    kubeAPIServerOverride: \"${KUBERNETES_SVC_EP_ADDR}\"|g" build/yamls/antrea.yml build/yamls/antrea-windows.yml
+        sed -i "s|.*kubeAPIServerOverride: \"\"|    kubeAPIServerOverride: \"${KUBERNETES_SVC_EP_ADDR}\"|g" build/yamls/antrea.yml
```

Comment:
it's surprising to me that we don't set `proxyAll: true` in the else block.

there is also some redundancy with what we do in `build_and_deliver_antrea_windows_and_linux_docker_images`?

Reply:
function deliver_antrea_linux is only used in windows docker test, it should be removed.